mods/ts/rules/aircraft.yaml

```diff
@@ -66,7 +66,7 @@ ORCA:
 		Cost: 1000
 	Tooltip:
 		Name: Orca Fighter
-		Description: Helicopter Gunship with Missiles.\n  Strong vs Buildings, Tanks\n  Weak vs Infantry
+		Description: Helicopter gunship with dual missile launchers.\n  Strong vs Buildings, Vehicles\n  Weak vs Infantry
```

Comment:
How about "Fast assault gunship armed with dual missile launchers" ?
I don't like the use of "helicopter", because they aren't, so I did some reading [here](http://cnc.wikia.com/wiki/Orca_Fighter), [here](http://cnc.wikia.com/wiki/Orca_aircraft) and [here](http://gaming.wikia.com/wiki/Units_%28Command_%26_Conquer:_Tiberian_Sun%29).